cloud/local-storage-pod.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: localstorage
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    command: ["sleep",  "300"]
+    resources:
+      requests:
+        memory: 10Mi
+    volumeMounts:
+    - name: cache-volume
+      mountPath: /cache
+  volumes:
+  - name: cache-volume
+    emptyDir: {}
```

Comment:
This is an emptyDir volume. @chao007 Can you please assist with that by providing a local volume example?

Reply:
OK, I took a look of https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md, this is valid, will merge